### PR TITLE
DROOLS-4425 : Migration from GDST to Excel for attributes and simple columns

### PR DESCRIPTION
**Pull request by Rikkola**
https://issues.jboss.org/browse/DROOLS-4425

https://github.com/kiegroup/drools/pull/2499
https://github.com/kiegroup/drools-wb/pull/1210

### CODE REVIEW

**Comment by Rikkola** [comment]
@manstis @jomarko This too could use reviews please.

File: drools-workbench-models/drools-workbench-models-guided-dtable/src/main/java/org/drools/workbench/models/guided/dtable/shared/util/ColumnUtilitiesBase.java [hunk 1]
```diff
@@ -0,0 +1,490 @@
...
+    protected final GuidedDecisionTable52 model;
+    private boolean respectLists;
+
+    public ColumnUtilitiesBase(final GuidedDecisionTable52 model) {
+        this(model,
+             true);
+    }
+
+    public ColumnUtilitiesBase(final GuidedDecisionTable52 model,
+                               final boolean respectLists) {
```
Comment (Rikkola) [inline drools-workbench-models/drools-workbench-models-guided-dtable/src/main/java/org/drools/workbench/models/guided/dtable/shared/util/ColumnUtilitiesBase.java:59]: This class was moved here as it was. Only change is this boolean that was added.

**Comment by Rikkola** [comment]
@manstis actually @jomarko has some changes that might change the codes. Perhaps best to wait for them before reviewing.

**Review by jomarko — COMMENTED** [review]
No comments to code, manual check starting right now.

**Comment by jomarko** [comment]
@Rikkola My changes will be "intrusive", I think I will create PR, but better to merge after 7.5

**Review by jomarko — APPROVED** [review]

**Comment by Rikkola** [comment]
@manstis ok, this would be ready for your reviews thank you.

File: drools-workbench-models/drools-workbench-models-guided-dtable/src/main/java/org/drools/workbench/models/guided/dtable/shared/util/ColumnUtilitiesBase.java [hunk 2]
```diff
@@ -0,0 +1,490 @@
...
+        this.model = PortablePreconditions.checkNotNull("model",
+                                                        model);
+
+        this.respectLists = PortablePreconditions.checkNotNull("respectLists",
```
Comment (manstis) [inline drools-workbench-models/drools-workbench-models-guided-dtable/src/main/java/org/drools/workbench/models/guided/dtable/shared/util/ColumnUtilitiesBase.java:63]: Not sure you can have a `null` _primitive_ type ;-)

**Review by manstis — APPROVED** [review]

File: drools-workbench-models/drools-workbench-models-guided-dtable/src/main/java/org/drools/workbench/models/guided/dtable/shared/util/ColumnUtilitiesBase.java (hunk 2 above)
Comment (Rikkola) [reply drools-workbench-models/drools-workbench-models-guided-dtable/src/main/java/org/drools/workbench/models/guided/dtable/shared/util/ColumnUtilitiesBase.java:63]: Hehe. Copy pasted the check to everything.

**Comment by Rikkola** [comment]
@kiegroup/gatekeepers Please merge these two.

**Pull request merged**